### PR TITLE
chore: bump aerogear sdk versions

### PR DESCRIPTION
**Pull request by darahayes**
##### Checklist

- [x] `npm test` passes
- [x] `npm run build` works
- [x] tests are included
- [x] documentation is changed or added

### CODE REVIEW

**Comment by wtrocki** [comment]
@darahayes Approving but it looks like keycloak update did not went smooth due to their breaking their promise types into native promise.

**Comment by darahayes** [comment]
@wtrocki I will do some checks to see if it needs to be fixed in the showcase or in the SDK itself

File: package.json [hunk 1]
```diff
@@ -16,13 +16,13 @@
   },
   "private": true,
   "dependencies": {
-    "@aerogear/app": "2.6.3",
```
Comment (wtrocki) [inline package.json]: 2.6 to 2.8? Why?

Comment (darahayes) [reply package.json]: A couple of releases of the JavaScript SDKs have been made and the showcase hasn't been updated accordingly.

Comment (wtrocki) [reply package.json]: Wow. We should not let this happen

**Comment by wtrocki** [comment]
Verified by running app

**Pull request merged**